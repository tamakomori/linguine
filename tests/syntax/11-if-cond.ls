func main() {
    if (0) {
        print("0");
    }
    if (1) {
        print("1");
    }
    if (0 == 0) {
        print("0 == 0");
    }
    if (0 == 1) {
        print("0 == 1");
    }
    if (1 == 0) {
        print("1 == 0");
    }
    if ("aaa" == "aaa") {
        print("aaa == aaa");
    }
    if ("aaa" == "bbb") {
        print("aaa == bbb");
    }
    if ("aaa" != "aaa") {
        print("aaa != aaa");
    }
    if ("aaa" != "bbb") {
        print("aaa != bbb");
    }
    if (0 != 0) {
        print("0 != 0");
    }
    if (0 != 1) {
        print("0 != 1");
    }
    if (1 != 0) {
        print("1 != 0");
    }
    if (0 >= 0) {
        print("0 >= 0");
    }
    if (0 >= 1) {
        print("0 >= 1");
    }
    if (1 >= 0) {
        print("1 >= 0");
    }
    if (0 > 0) {
        print("0 > 0");
    }
    if (0 > 1) {
        print("0 > 1");
    }
    if (1 > 0) {
        print("1 > 0");
    }
}
