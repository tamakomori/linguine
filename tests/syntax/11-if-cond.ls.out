1
0 == 0
aaa == aaa
aaa != bbb
0 != 1
1 != 0
0 >= 0
1 >= 0
1 > 0
